.gitignore

```diff
@@ -1,4 +1,5 @@
 .env
 *.env.*
-
+local.env
```

Comment:
You can just do `*.env` since it's local to you. You can even have a global `.gitignore` where things you ignore locally will be automatically ignored by each repo.

Reply:
i moved this to my global git ignore and removed it here. thank you for the tip